src/components/pages/Theme/Theme.jsx

```diff
@@ -297,6 +319,95 @@ export const Theme = () => {
           </div>
         </Segment>
       </div>
+
+      <div className='section'>
+        <Segment padded>
+          <Header as='h2' dividing>
+            Message
+          </Header>
+          <MessageExample header='Title' content={messageText} icon='alarm' />
+
+          <Header size='small' dividing>
+            Info
+          </Header>
+          <MessageExample content={messageText} info icon='info circle' />
+
+          <Header size='small' dividing>
+            Warning
+          </Header>
+          <MessageExample content={messageText} warning icon='warning sign' />
+
+          <Header size='small' dividing>
```

Comment:
I wanted to show that the styles are applied regardless of what prop is passed; eg. it can be `<Message positive />` or `<Message success />`. Hence, there're 2 alerts.
For the secondary variant I don't have this because it only has only 1 class name (`<Message color='green' />`).